### ISSUE TITLE
Volume buttons change media volume in open conversations without any media playing

### ISSUE THREAD

**Issue by hwinnemoe** [signalapp/Signal-Android#10130]
- [X] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [X] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Using the volume buttons on the device, the media volume is changed in open conversations without any media playing. The notification volume should be changed instead. This still works correctly in the conversation list.

### Steps to reproduce
- Open a conversation
- Try changing the volume of your device

**Actual result:** The media volume is changed.
**Expected result:** The notification volume is changed.

### Screenshots

(I can not make screenshots of this specific UI element)

### Device info

**Device:** Sony Xperia Z2, Android 
**Android version:** 6.0.1
**Signal version:** 4.74.3

### Link to debug log

[can be sent upon request]

**Comment by mfb** [comment]
Here's a screenshot:
![volume-bug](https://user-images.githubusercontent.com/3278/101710490-6dd92080-3a46-11eb-81f5-96682209e698.jpg)

**Comment by LeKoe** [comment]
Same problem here
Shiftphone 5me
Android 8.0
Signal Version 5.4.12

**Comment by mfb** [comment]
This is still a bug on current version. 

I suppose it could make sense to be able to change media volume before you play a media item, but the volume buttons change media volume even if I'm on a new thread that doesn't have any playable media at all.

**Comment by LeKoe** [comment]
Still a Bug. Is there anyone with an idea how to solve this?

**Comment by cody-signal** [comment]
@LeKoe are you still running Android 8? I think the volume adjustment varies based on OS version. Like on my device running Android 12, the volume rocker always adjust media volume regardless of where I am (home, signal, gmail, etc.)

**Comment by mfb** [comment]
I'm on Android 12 and I have the "Use Volume keys for media" setting turned off (it can be overridden by apps). The volume buttons control ringtone in home and most apps, except in e.g. Signal and VLC they control media volume. For VLC this makes sense as it's a media player, but not imho for Signal, unless there is an audio/video message playing or at least visible on screen (I suppose maybe you want to adjust media volume before tapping play?).

**Comment by cody-signal** [comment]
Thanks for the extra info @mfb. If you are in Signal app settings do the keys work as expected and the issue occurs only on Conversation List and Conversation screens?

This is likely due to the fact we start up a media session to help with voice note playback on these two screens and the phone thinks we are in a media session when we are not.

**Comment by mfb** [comment]
Yep the volume buttons work as expected (ringtone volume) on the Signal settings screen.

I find myself fairly often in the situation of wanting to control ringtone volume while on Signal conversation/conversation list screens (either to make sure I hear a notification or to quiet them down).  At this point I've gotten used to going back to home screen or some other app to control ringtone volume, but if there's any way to resolve the issue it would be a nice improvement.

**Comment by cody-signal** [comment]
Thanks for confirming, we'll look into ways to not start up the voice note service each time if possible.
